### ISSUE TITLE
Make Lunar Probe Landing a milestone for the manned landing?

### ISSUE THREAD

**Issue by peyre** [raceintospace/raceintospace#177]
As it is, the lunar probe landing is completely optional--I always skip it because it's just a milestone penalty to the manned lunar pass, and rarely causes me trouble.  However it makes a lot of sense that, before landing people on the Moon, you really would want to land a probe first.

Seems to me that if we really want to encourage people to do the landing probe, we should make it a milestone specifically for the manned lunar landing.  That, combined with not having enough LM points, would really be an obstacle for someone trying to cut corners.

**Comment by peyre** [comment]
Of course, that's if we _want_ to do that.  Personally I'm fine with the way things are, and changing it would force me to change my strategy in the game, and would make it harder to reach the Moon before the end of the decade.  But logically speaking, this does seem like a sound idea.

**Comment by rnyoakum** [comment]
Reviving an old issue: as I understand (see #246, #250) the Lunar Probe is a milestone for a Manned Lunar Landing, given how mission penalties work. Were you suggesting there should be an extra penalty associated with it, beyond the standard -3 Milestone penalty? If not, we can probably close this issue.

**Comment by peyre** [comment]
No.  Currently in the game (dating back to _Liftoff!_), the Lunar Probe Landing is a milestone for the Manned Lunar Pass.  It was designed as a linear thing, and that makes sense in a board game, but I think for our purposes it would be better if it were a milestone for the lunar landing.  After all, what does it matter for the lunar pass if you haven't landed a probe?  The lunar pass doesn't land.  But it sure does matter for the lunar landing if you haven't landed a probe yet.

**Comment by peyre** [comment]
An added (though very small) benefit of making the probe landing a milestone for the manned landing is that we'd avoid seeing it show like in this screenshot.  Here, I've skipped the probe landing (as I always do) and flew the manned lunar pass.  It's a little odd to see the probe landing show, though I understand why: this screen shows you getting credit for anything you've accomplished, including anything you've skipped.

![Manned Lunar Pass](https://user-images.githubusercontent.com/3118014/67623746-bb141780-f7dd-11e9-9ea5-24c36a5f5cb0.png)

**Comment by peyre** [comment]
The more I think about this issue, the better an idea I think it would be, in terms of game balance, even if it gets the game a bit further from its roots in _Liftoff!_.  It would increase pressure on the player to do at least this _or_ the LM points before attempting a landing.  As it is, both can be skipped most of the time.

**Comment by rnyoakum** [comment]
I had thought that the Lunar Probe Landing milestone wouldn't be awarded if you hadn't actually performed a Lunar Probe Landing, but your previous post shows otherwise. If I remember correctly, the mission data files list all of the milestones the mission (should) satisfy, so it was surprising to learn the milestones chained like this. But it's possible I was getting my Prestige Achievements and Milestones confused. Unfortunately, this creates a conundrum:

1. If the Lunar Probe Landing is the last item before Manned Lunar Landing, launching a lunar landing probe would be very hard to do early, but could chain you all the way through Manned Lunar Pass and Manned Lunar Orbital. The least of the problems with this is that it doesn't make any sense. It also makes a Lunar Probe Landing much harder to do early on. Most concerning: given that it's **much** lower risk than a manned mission (there is no crew and none of the steps associated with them), that would make it an extremely potent - if risky - way to get ahead and clear out milestone penalties. If it's giving Lunar Pass / Lunar Orbit prestige as well, it also becomes extremely lucrative.
2. If the Lunar Probe Landing comes before Manned Lunar Pass, it's creating a milestone penalty for the Lunar Pass (which doesn't make sense) and becomes irrelevant once the Lunar Pass is achieved, which makes it an expensive proposition. It's faster than Lunar Passes / Flybys and gives a modest prestige award, but there's the expense of buying the program and researching it.

At the moment, I think the problems from (1) with a monster Lunar Probe Landing are greater, at least if Prestige is being awarded. Maybe that's balanced by a slower manned program, losing valuable Manned prestige. The big problem is that neither (1) nor (2) are reasonable because they don't make *sense*.

The correct solution is that Milestones shouldn't be a strictly linear path. For the most part, that approach works (because pushing further into space is the nature of the thing) but the Probes are something of an exception. I haven't tested recently enough to remember if a Lunar Flyby gives credit for Manned Suborbital & Earth Orbital, but it definitely shouldn't. However, a Manned Lunar Pass should give credit for a Lunar Flyby.

I haven't worked with Milestones much, but I think there is the structure in place for a non-linear build. If that's the case, the code for awarding the milestones could be adjusted, and would eliminate most of the issues with (1). Then Lunar Probe Landing could be moved to the back of the line, a prerequisite only for the Manned Lunar Landing. At that point it would still be worth doing - a 3 point penalty on every stage of a Manned Lunar Landing adds up fast - but wouldn't risk game balance elsewhere.

**Comment by peyre** [comment]
Ohhh, I hadn't thought of that.  Of course the milestone path in (BA)RIS is linear, meaning that if we put it right before the landing, someone could fly the Lunar Probe Landing _instead of_ the Lunar Pass and Lunar Orbital.  That would be a huge exploit, and would be one of those cures that's worse than the disease.

Yes, the ideal would be for the probe landing to be a milestone (or whatever) for just the Lunar Landing itself.  It might function similarly to the LM Points system, except it should be a -3 for all steps of the mission, or perhaps a -9 on the lunar steps of the landing, like with LM Points (but doing one probe landing would remove the -9 altogether).

**Comment by peyre** [comment]
If we were to do this, I wonder if we might also do something to make the Unmanned Lunar Pass and Orbital more useful.  Maybe they could be a milestone step for the Manned Lunar Pass (that is, you could use either to satisfy the milestone requirement).  I know it's outside the way we normally do things - milestones risk your spacepeople and gain/lose you prestige...but this could be different.  The Soviets flew several unmanned capsules around the Moon but didn't get any credit for that, yet it seems logical you'd want to fly a ship out there once before risking lives.  This would make those missions good for something other than dummy-testing your kickers for Gemini and XMS-2 flights.

**Comment by peyre** [comment]
Another option might be something like, if you skip the landing probe, you take a 20% safety penalty on the landing or EVA steps of the lunar landing.  That would be simpler than implementing another LM Points system or creating a special milestone penalty for the landing.

**Comment by peyre** [comment]
If we should ever implement something to make the probe landing more crucial to the success of the landing, I'd suggest also that we bump up the lunar recon that you get from a lunar orbital.  As it is, you get 5% from either a pass or an orbital, which doesn't seem right since the orbital spends a lot more time around the Moon.  I'd suggest having the orbital give you 10%, but not without doing something to make the probe landing more important, since just giving more recon for the orbital actually makes the probe landing slightly _less_ important, which I don't want to do.

**Comment by hweimer** [comment]
One could limit the Max Recon level that can be reached without probes to something like 70%.

**Comment by peyre** [comment]
Well that's an idea.  It would seem a shame to not get recon from lunar passes and orbitals though, if you're at 70% already.

**Comment by peyre** [comment]
Ok, this morning I had time and money so I decided to fly the probe landing again, literally for the first time in years.  FIVE ATTEMPTS IN A ROW FAILED.  Every Single attempt.  It held up my lunar flyby so long, I finally went ahead with it with the milestone penalty. 
**45MB** to prototype + ≥**40MB** to research (with tech transfer from Ranger) + **88MB** to fly (4 additional probes, plus 5 Titans) = **173MB**, all for nothing.
If we should decide to make the probe more critical to the lunar landing, I really feel we _must_ do something to make it cheaper and/or more likely to succeed, provided you've done successful flyby(s).

![SurveyorFailure1](https://user-images.githubusercontent.com/3118014/119233059-0edf0380-badc-11eb-86a8-17cff6062622.png)
![SurveyorFailure3](https://user-images.githubusercontent.com/3118014/119233061-0edf0380-badc-11eb-8153-2b720d75c2c8.png)

**Comment by smudog** [comment]
Surveyor was a giant pain in real life.  It was hard coupled to the complex Centaur program, there were HQ-JPL battles, JPL was distracted by the existential crisis that was Ranger, and in the end it went four times over the initial budget and two years late, after significant descopes.  While it clearly proved the point of landing (including disproving Tom Gold on the structure of the moons surface), I don't know that any technology transferred to the LEM program.  Both Centaur and the closed loop landing tech were tremendously valuable, but only after July 20, 1969.

**Comment by peyre** [comment]
Oh, man!  I hate being caught between realism and gameplay.

**Comment by smudog** [comment]
On the gameplay I think a penalty against the lunar landing phase itself for not having the would be appropriate (that would represent the real unknown of the moon not actually being something you can land on) - I don't know how implementable that would be.  I think also making it cheaper to buy (as apposed to research) would be appropriate.

**Comment by peyre** [comment]
Raising its Max R&D would also be a good idea, I think.

**Comment by hweimer** [comment]
Given the historical context provided by @smudog, I'm not sure that fhis is really a problem. If the Surveyor program was such a pain in the neck, well then you maybe simply shouldn't do it and live with the milestone penalty for the manned lunar pass. It never occured to me that this was a viable route until I read this bug report (after all, you skip an unmanned milestone for which you have to pay during a manned mission), but now I am convinced that this is usually a risk worth taking. Just think how many dummy tests raising the safety you can fly for 192 MB.

PS: I think it makes some sense to make the lunar probe landing a milestone for the lunar pass. Landing a probe requires quite a bit more accurate navigation than a simple flyby, and that's certainly something that is crucial for a successful lunar pass as well.

PPS: It's not quite correct to say that Surveyor disproved Thomas Gold. On the contrary, his prediction that the lunar surface is covered with a layer of dust turned out to be spot-on. He just wasn't sure how thick this layer was and therefore it could have been too risky to land on the moon. But these kind of risks are not really modeled in RIS (and I also think that one could have gathered the necessary information about the lunar surface by a beefed-up impactor mission).

**Comment by peyre** [comment]
I think I got the idea of skipping the probe landing from the _Buzz Aldrin's Race Into Space Companion_.  Somewhere in there it mentions that the probe landing is really optional.

**Comment by peyre** [comment]
Found it.  It's on page 265.
